src/uct/api/v2/uct_v2.h

```diff
@@ -68,11 +69,17 @@ enum uct_perf_attr_field {
     /** Enables @ref uct_perf_attr_t::remote_memory_type */
     UCT_PERF_ATTR_FIELD_REMOTE_MEMORY_TYPE = UCS_BIT(2),
 
+    /** Enables @ref uct_perf_attr_t::local_sys_device */
+    UCT_PERF_ATTR_FIELD_LOCAL_SYS_DEVICE   = UCS_BIT(3),
+
+    /** Enables @ref uct_perf_attr_t::remote_sys_device */
+    UCT_PERF_ATTR_FIELD_REMOTE_SYS_DEIVCE  = UCS_BIT(4),
```

Comment:
@yosefe for backward compatibility it supposed to be appended to the end.

Reply:
@shamisp in general yes, but is is (unreleased) _v2 API

Reply:
So, this is just preferred order ?

Reply:
yes, right after memory types